Auto update failing for kakoune

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the kakoune package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating kakoune [Current version: 2022.10.31]
INFO: package being updated to 2023.07.29.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  554k  100  554k    0     0  1525k      0 --:--:-- --:--:-- --:--:-- 1525k

You are about to commit these changes:

--------------------
diff --git a/packages/kakoune/build.sh b/packages/kakoune/build.sh
index a50dbc5eb..c064b1bf9 100644
--- a/packages/kakoune/build.sh
+++ b/packages/kakoune/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/mawww/kakoune
 TERMUX_PKG_DESCRIPTION="Code editor heavily inspired by Vim"
 TERMUX_PKG_LICENSE="Unlicense"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2022.10.31"
+TERMUX_PKG_VERSION="2023.07.29"
 TERMUX_PKG_SRCURL=https://github.com/mawww/kakoune/releases/download/v$TERMUX_PKG_VERSION/kakoune-$TERMUX_PKG_VERSION.tar.bz2
-TERMUX_PKG_SHA256=fb317b62c9048ddc7567fe83dfc409c252ef85778b24bd2863be2762d4e4e58b
+TERMUX_PKG_SHA256=6a73782f3f791c4eede7ffbbfeec24370b3b62e519301120912736ac5bd789e2
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(main/kakoune): 2023.07.29


Not committing to Git!

INFO: Trying to build package.
INFO: Package kakoune exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
Creating new container...
Unable to find image 'ghcr.io/termux/package-builder:latest' locally
latest: Pulling from termux/package-builder
9d19ee268e0d: Pulling fs layer
0dfc5f884292: Pulling fs layer
f22bf3c94a25: Pulling fs layer
e19f2fd4c7c3: Pulling fs layer
43d5e7d249b7: Pulling fs layer
062ef83e7b7c: Pulling fs layer
e19f2fd4c7c3: Waiting
43d5e7d249b7: Waiting
062ef83e7b7c: Waiting
f22bf3c94a25: Verifying Checksum
f22bf3c94a25: Download complete
0dfc5f884292: Verifying Checksum
0dfc5f884292: Download complete
9d19ee268e0d: Verifying Checksum
9d19ee268e0d: Download complete
e19f2fd4c7c3: Verifying Checksum
e19f2fd4c7c3: Download complete
062ef83e7b7c: Verifying Checksum
062ef83e7b7c: Download complete
9d19ee268e0d: Pull complete
0dfc5f884292: Pull complete
f22bf3c94a25: Pull complete
e19f2fd4c7c3: Pull complete
43d5e7d249b7: Verifying Checksum
43d5e7d249b7: Download complete
43d5e7d249b7: Pull complete
062ef83e7b7c: Pull complete
Digest: sha256:2a47ca0b587598ab671a3612b7b301248e1a323902526183ad2709feccff44f6
Status: Downloaded newer image for ghcr.io/termux/package-builder:latest
e146873936b768d624cc0cb1ac0621b5bfd53b3e64505f40b45eddb69bffab17
gpg: key B0076E490B71616B: 17 signatures not checked due to missing keys
gpg: key B0076E490B71616B: public key "Henrik Grimler <henrik@grimler.se>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: no ultimately trusted keys found
gpg (GnuPG) 2.2.27; Copyright (C) 2021 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

gpg: key 5A897D96E57CF20C: public key "Termux Releases (Termux automatic builds) <contact@termux.dev>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   1  signed:   1  trust: 0-, 0q, 0n, 0m, 0f, 1u
gpg: depth: 1  valid:   1  signed:   0  trust: 1-, 0q, 0n, 0m, 0f, 0u
gpg: next trustdb check due at 2024-08-25
gpg (GnuPG) 2.2.27; Copyright (C) 2021 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

termux - building kakoune for arch aarch64...
Downloading https://github.com/termux/termux-elf-cleaner/releases/download/v2.1.1/termux-elf-cleaner
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 63912  100 63912    0     0   190k      0 --:--:-- --:--:-- --:--:--  190k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  30945      0 --:--:-- --:--:-- --:--:-- 30903
100 13134  100 13134    0     0  30931      0 --:--:-- --:--:-- --:--:-- 30903
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1574      0 --:--:-- --:--:-- --:--:--  1574
gpg: Signature made Sat 29 Jul 2023 12:36:59 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: checking the trustdb
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   2  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 2u
gpg: next trustdb check due at 2024-08-25
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  3 1523k    3 47004    0     0   110k      0  0:00:13 --:--:--  0:00:13  110k
100 1523k  100 1523k    0     0  3625k      0 --:--:-- --:--:-- --:--:-- 3619k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  30467      0 --:--:-- --:--:-- --:--:-- 30480
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2123      0 --:--:-- --:--:-- --:--:--  2125
gpg: Signature made Fri 21 Jul 2023 01:56:11 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/stable/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 54348  100 54348    0     0   128k      0 --:--:-- --:--:-- --:--:--  128k
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  27113      0 --:--:-- --:--:-- --:--:-- 27163
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2026      0 --:--:-- --:--:-- --:--:--  2026
gpg: Signature made Fri 28 Jul 2023 11:27:29 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  413k  100  413k    0     0  1046k      0 --:--:-- --:--:-- --:--:-- 1047k
Downloading dependency libc++@25c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libc++/libc++_25c_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  216k  100  216k    0     0  2080k      0 --:--:-- --:--:-- --:--:-- 2098k
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/mawww/kakoune/releases/download/v2023.07.29/kakoune-2023.07.29.tar.bz2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  554k  100  554k    0     0  1917k      0 --:--:-- --:--:-- --:--:-- 1917k
Applying ndk-patch: bits-struct_file.h.patch
Applying ndk-patch: dirent.h.patch
Applying ndk-patch: grp.h.patch
Applying ndk-patch: linux-fcntl.h.patch
Applying ndk-patch: paths.h.patch
Applying ndk-patch: pwd.h.patch
Applying ndk-patch: redefine-TCSAFLUSH.patch
Applying ndk-patch: semaphore.h.patch
Applying ndk-patch: stdio.h.patch
Applying ndk-patch: stdlib.h.patch
Applying ndk-patch: sys-cdefs.h.patch
Applying ndk-patch: syslog.patch
Applying ndk-patch: unistd.h.patch
Applying ndk-patch: utmp.h.patch
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libicu.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libnativehelper.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libicu.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libnativehelper.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/32/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libicu.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libnativehelper.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libz.so'
Applying patch: src-cs-path-compat.patch
Applying patch: src-main.cc.patch
make: Entering directory '/home/builder/.termux-build/kakoune/src/src'
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .alias_registry.opt.d -c -o .alias_registry.opt.o alias_registry.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .assert.opt.d -c -o .assert.opt.o assert.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .backtrace.opt.d -c -o .backtrace.opt.o backtrace.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .buffer.opt.d -c -o .buffer.opt.o buffer.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .buffer_manager.opt.d -c -o .buffer_manager.opt.o buffer_manager.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .buffer_utils.opt.d -c -o .buffer_utils.opt.o buffer_utils.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .changes.opt.d -c -o .changes.opt.o changes.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .client.opt.d -c -o .client.opt.o client.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .client_manager.opt.d -c -o .client_manager.opt.o client_manager.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .color.opt.d -c -o .color.opt.o color.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .command_manager.opt.d -c -o .command_manager.opt.o command_manager.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .commands.opt.d -c -o .commands.opt.o commands.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .completion.opt.d -c -o .completion.opt.o completion.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .context.opt.d -c -o .context.opt.o context.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .display_buffer.opt.d -c -o .display_buffer.opt.o display_buffer.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .env_vars.opt.d -c -o .env_vars.opt.o env_vars.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .event_manager.opt.d -c -o .event_manager.opt.o event_manager.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .exception.opt.d -c -o .exception.opt.o exception.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .face_registry.opt.d -c -o .face_registry.opt.o face_registry.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .file.opt.d -c -o .file.opt.o file.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .hash.opt.d -c -o .hash.opt.o hash.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .hash_map.opt.d -c -o .hash_map.opt.o hash_map.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .highlighter.opt.d -c -o .highlighter.opt.o highlighter.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .highlighter_group.opt.d -c -o .highlighter_group.opt.o highlighter_group.cc
aarch64-linux-android-clang++  -I/data/data/com.termux/files/usr/include  -fstack-protector-strong -Oz -O3 -pedantic -std=c++2a -g -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-address -frelaxed-template-template-args -Wno-ambiguous-reversed-operator -MD -MP -MF .highlighters.opt.d -c -o .highlighters.opt.o highlighters.cc
In file included from highlighters.cc:1:
In file included from ./highlighters.hh:4:
In file included from ./color.hh:4:
In file included from ./exception.hh:4:
In file included from ./string.hh:6:
In file included from ./units.hh:8:
In file included from /home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/compare:51:
In file included from /home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/array:110:
/home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/iterator:783:41: error: call to deleted function 'addressof'
    pointer  operator->() const {return _VSTD::addressof(operator*());}
                                        ^~~~~~~~~~~~~~~~
/home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/__config:795:15: note: expanded from macro '_VSTD'
#define _VSTD std::_LIBCPP_ABI_NAMESPACE
              ^
./display_buffer.hh:145:53: note: in instantiation of member function 'std::reverse_iterator<std::move_iterator<std::__wrap_iter<const Kakoune::DisplayAtom *>>>::operator->' requested here
            m_range.end = std::max(m_range.end, last->end());
                                                    ^
highlighters.cc:1853:32: note: in instantiation of function template specialization 'Kakoune::DisplayLine::insert<std::move_iterator<std::__wrap_iter<const Kakoune::DisplayAtom *>>>' requested here
            auto it = line_it->insert(
                               ^
/home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/type_traits:664:27: note: candidate function [with _Tp = Kakoune::DisplayAtom] has been explicitly deleted
template <class _Tp> _Tp* addressof(const _Tp&&) noexcept = delete;
                          ^
/home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/type_traits:605:1: note: candidate function [with _Tp = const Kakoune::DisplayAtom]
addressof(_Tp& __x) _NOEXCEPT
^
In file included from highlighters.cc:1:
In file included from ./highlighters.hh:4:
In file included from ./color.hh:4:
In file included from ./exception.hh:4:
In file included from ./string.hh:6:
In file included from ./units.hh:8:
In file included from /home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/compare:51:
In file included from /home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/array:110:
/home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/iterator:783:41: error: calling a private constructor of class 'std::__wrap_iter<const Kakoune::DisplayAtom *>'
    pointer  operator->() const {return _VSTD::addressof(operator*());}
                                        ^
/home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/__config:795:15: note: expanded from macro '_VSTD'
#define _VSTD std::_LIBCPP_ABI_NAMESPACE
              ^
/home/builder/.termux-build/_cache/android-r25c-api-24-v2/bin/../sysroot/usr/include/c++/v1/iterator:1530:60: note: declared private here
    _LIBCPP_INLINE_VISIBILITY _LIBCPP_CONSTEXPR_IF_NODEBUG __wrap_iter(iterator_type __x) _NOEXCEPT : __i(__x) {}
                                                           ^
2 errors generated.
make: *** [Makefile:104: .highlighters.opt.o] Error 1
make: *** Waiting for unfinished jobs....
make: Leaving directory '/home/builder/.termux-build/kakoune/src/src'
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2023-07-29 12:14:07 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/5700279725">5700279725</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/kakoune/build.sh
+++ b/packages/kakoune/build.sh
@@ -2,13 +2,21 @@ TERMUX_PKG_HOMEPAGE=https://github.com/mawww/kakoune
 TERMUX_PKG_DESCRIPTION="Code editor heavily inspired by Vim"
 TERMUX_PKG_LICENSE="Unlicense"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2022.10.31"
+TERMUX_PKG_VERSION="2023.07.29"
 TERMUX_PKG_SRCURL=https://github.com/mawww/kakoune/releases/download/v$TERMUX_PKG_VERSION/kakoune-$TERMUX_PKG_VERSION.tar.bz2
-TERMUX_PKG_SHA256=fb317b62c9048ddc7567fe83dfc409c252ef85778b24bd2863be2762d4e4e58b
+TERMUX_PKG_SHA256=6a73782f3f791c4eede7ffbbfeec24370b3b62e519301120912736ac5bd789e2
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_EXTRA_MAKE_ARGS=" -C src debug=no "
+
+termux_step_pre_configure() {
+	# This patch should be no longer needed when NDK bumps to r26
+	local _file="$TERMUX_PKG_BUILDER_DIR/src-display_buffer.hh.patch"
+	if [ -f $_file && "$TERMUX_NDK_VERSION" != 25c ]; then
+		termux_error_exit "Please remove $_file"
+	fi
+}
 
 termux_step_create_debscripts() {
 	cat <<- EOF > ./postinst

--- a/packages/kakoune/src-display_buffer.hh.patch
+++ b/packages/kakoune/src-display_buffer.hh.patch
@@ -1,0 +1,32 @@
+--- a/src/display_buffer.hh
++++ b/src/display_buffer.hh
+@@ -137,6 +137,21 @@
+     template<typename It>
+     iterator insert(iterator pos, It beg, It end)
+     {
++#ifdef __ANDROID__
++        auto first = beg;
++        for (; first != end; ++first)
++            if (first->has_buffer_range())
++                break;
++        if (first != end) {
++            auto last = end - 1;
++            for (; last >= first; --last)
++                if (last->has_buffer_range())
++                    break;
++            m_range.begin = std::min(m_range.begin, first->begin());
++            m_range.end = std::max(m_range.end, last->end());
++        }
++        return m_atoms.insert(pos, beg, end);
++#else
+         auto has_buffer_range = std::mem_fn(&DisplayAtom::has_buffer_range);
+         if (auto first = std::find_if(beg, end, has_buffer_range); first != end)
+         {
+@@ -145,6 +160,7 @@
+             m_range.end = std::max(m_range.end, last->end());
+         }
+         return m_atoms.insert(pos, beg, end);
++#endif
+     }
+ 
+     DisplayLine extract(iterator beg, iterator end);

